src/test/java/com/microsoft/azure/servicebus/jms/aad/QueueTest.java

```diff
@@ -1,6 +1,6 @@
 package com.microsoft.azure.servicebus.jms.aad;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
```

Comment:
Not for here, but I would make the title for this as 
Update qpid.jms.client and other dependencies for compliance.